src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -202,6 +203,24 @@
         - name: --aks-custom-headers
           type: string
           short-summary: Send custom headers. When specified, format should be Key1=Value1,Key2=Value2
+        - name: --appgw-name
+          type: string
+          short-summary: Name of the application gateway to create/use in the node resource group
+        - name: --appgw-subnet-prefix
+          type: string
+          short-summary: Subnet Prefix to use for a new subnet created to deploy the Application Gateway
+        - name: --appgw-id
+          type: string
+          short-summary: Resource Id of an existing Application Gateway to use with AGIC
+        - name: --appgw-subnet-id
```

Comment:
It would be better if we can support name or ID. Just a kind reminder. It's not mandatory.
https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#supporting-name-or-id-parameters

Reply:
name and id are supported for 2 different scenarios:
id is used for BYO (bring your iown) gateway.
name is for when gateway needs to be created.